src/models/index.js

```diff
@@ -991,7 +991,22 @@ module.exports = cooler => {
         // We default to `null` in case something goes wrong and we receive a
         // private message with no `recps` somehow. This adds a layer of security
         // because SSB-DB won't let us publish a message with `{ recps: null }`.
```

Comment:
This comment doesn't match the code now - it defaults to `[]` instead of `null`.  Is it worth checking for an empty array after all this and setting it to `null` in that case?

Reply:
Thanks, I'll fix that comment! `[]` is fine too, SSB won't publish a message with `recps` unless `recps` is an array with >= 1 recipient which has to be a valid SSB feed ID. I'll add some more security here anyway, just in case.